### PR TITLE
AHWR-403 - Ensuring we pass on the claim reference in the submission of the clai…

### PR DESCRIPTION
**Pull request by RobCatton**
…m to the application repo

### CODE REVIEW

File: app/routes/endemics/check-answers.js [hunk 1]
```diff
@@ -584,14 +584,15 @@ const postHandler = {
         }
       }, request.logger)
 
-      setEndemicsClaim(request, 'reference', claim.reference)
-      setEndemicsClaim(request, 'amount', claim.data?.amount)
+      setEndemicsClaim(request, 'reference', claim.dataValues.reference)
```
Comment (aaroncarroll82) [inline app/routes/endemics/check-answers.js:587]: Does this need to be changed? The other end is not changing what it returns in response is it?

File: test/integration/narrow/routes/endemics/check-answers.test.js [hunk 2]
```diff
@@ -533,7 +534,7 @@ describe('Check answers test', () => {
         getEndemicsClaimMock.mockImplementation(() => {
           return {
             typeOfLivestock: 'pigs',
-            typeOfReview: 'review',
+            typeOfReview: 'R',
```
Comment (aaroncarroll82) [inline test/integration/narrow/routes/endemics/check-answers.test.js:537]: How come this has changed?

Comment (RobCatton) [reply test/integration/narrow/routes/endemics/check-answers.test.js:537]: Its largely irrelevant, but typeOfReview is R or E - you can see in the other test similar to this one that it has typeOfReview E

File: app/routes/endemics/check-answers.js (hunk 1 above)
Comment (RobCatton) [reply app/routes/endemics/check-answers.js:587]: I'm not hugely confident but I think it has been incorrect for a while now. It was setting the values using claim.reference and claim.data?.amount, but the 'claim' object which is returned from the application service is the DB Model, which has all the data contained inside dataValues.

Comment (RobCatton) [reply app/routes/endemics/check-answers.js]: Fixed!

**Review by aaroncarroll82 — APPROVED** [review]

**Pull request merged**